.babelrc

```diff
@@ -1,14 +1,19 @@
 {
   "presets": [
-    ["es2015", { "loose": true }],
-    "stage-1",
+    ["env", {
+      "targets": {
+        "ie": 9,
+        "uglify": true
```

Comment:
what's this do?

Reply:
ensures it compiles code down to at least what uglify can parse, e.g es5

Reply:
isn't that what no-options `env` does? just target ES5, like with `latest`? i would just do plain `env` then, with `loose`.

Reply:
i'm copying the CRA config here, they generally have good reasons for the settings

Reply:
fair enough. let's use `loose` though?

Reply:
good call